src/index.html

```diff
@@ -6,12 +6,253 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta 
+    http-equiv="X-UA-Compatible" 
+    content="ie=edge"
+    >
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link 
+    rel="stylesheet" 
+    href="./style.css"
+    >
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post"
+      class="form"
+      >
+
+      <fieldset class="group">
+        <legend>
+        Personal information
+        </legend>
+        <div class="simple">
+          <label>
+            Surname:
+            <input 
+            type="text" 
+            name="sname" 
+            required 
+            autocomplete="off"
+            >
+          </label> 
+        </div>
+        <div class="line"></div>
+        <div class="simple">
+          <label>
+            Name:
+            <input 
+            type="text" 
+            name="name" 
+            required 
+            autocomplete="off"
+            >
+          </label>
+        </div>
+        
+        <div class="simple">
+          <label>
+            How old are You?
+            <input 
+            type="number" 
+            name="old" 
+            min="1" 
+            max="100" 
+            value="12" 
+            required
+            >
+          </label>
+        </div>
+        
+        <div class="simple">
+          <label>
+            Full date of birth:
+            <input 
+            type="date" 
+            name="birth" 
+            required
+            >
+          </label>
+        </div>
+        <div class="line"></div>
+        <div>
+          <label>
+            I accept the term of the agreement:
+            <input 
+            type="checkbox" 
+            name="accept" 
+            required
+            >
+          </label>
+        </div>
+      </fieldset>
+    
+
+    
+      <fieldset class="group">
```

Comment:
is too big distance between siblings
```suggestion
      </fieldset>

      <fieldset class="group">
```